packages/node/src/plugins/NodeFederationPlugin.ts

```diff
@@ -105,13 +124,15 @@ export const generateRemoteTemplate = (
     return remote;
```

Comment:
The function 'parseRemoteSyntax' does not handle the case when the remote parameter is null or undefined. This could lead to unexpected behavior or errors. Add a check to handle such cases.